tests/countries/test_vanuatu.py

```diff
@@ -0,0 +1,55 @@
+#  python-holidays
+#  ---------------
+#  A fast, efficient Python library for generating country, province and state
+#  specific sets of holidays on the fly. It aims to make determining whether a
+#  specific date is a holiday as fast and flexible as possible.
+#
+#  Authors: dr-prodigy <dr.prodigy.github@gmail.com> (c) 2017-2023
+#           ryanss <ryanssdev@icloud.com> (c) 2014-2017
+#  Website: https://github.com/dr-prodigy/python-holidays
+#  License: MIT (see LICENSE file)
+
+from holidays.countries.vanuatu import Vanuatu, VU, VTU
+from tests.common import TestCase
+
+
+class TestVanuatu(TestCase):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass(Vanuatu)
+
+    def test_country_aliases(self):
+        self.assertCountryAliases(Vanuatu, VU, VTU)
+
+    def test_no_holidays(self):
+        self.assertNoHolidays(Vanuatu(years=1980))
+
+    def test_fater_lina_day(self):
+        name = "Father Lini Day"
+        self.assertHolidayName(name, (f"{year}-2-21" for year in range(1991, 2050)))
+        self.assertNoHolidayName(name, Vanuatu(years=range(1981, 1991)))
+        self.assertNoHoliday(
+            f"{year}-2-21" for year in set(range(1981, 1991)).difference({1982, 1984, 2085})
+        )
```

Comment:
```suggestion
    def test_fater_lini_day(self):
        name = "Father Lini Day"
        self.assertHolidayName(name, (f"{year}-02-21" for year in range(1991, 2050)))
        self.assertNoHolidayName(name, Vanuatu(years=range(1981, 1991)))
        self.assertNoHoliday(f"{year}-02-21" for year in range(1981, 1991))
```
I don't see any reasons for exceptions for 1982, 1984.

Reply:
I copied a code from another country and forgot to change it. Sorry